### PR TITLE
[homework #6] - js basics 1 by German Macchia

### PR DESCRIPTION
**Pull request by GermanMacchia**
Hello @VitalikMelnyk!
Mike told us to make a pull request of our homework! 
I would push the last excercises in the next hours and continue with next tasks!

### CODE REVIEW

File: homeworks/german-macchia/06-basics-1/basics-1.js [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+
+  for (let i = 0; i < loops; i++) {
+    arr[i] = fn(a0[i], a1[i]);
+  }
+
+  return arr;
+}
+
+// katas 12 https://www.codewars.com/kata/filter-the-number
+let FilterString = function (value) {
```
Comment (VitalikMelnyk) [inline homeworks/german-macchia/06-basics-1/basics-1.js:267]: Use camelCase for naming of functions

**Comment by VitalikMelnyk** [comment]
@GermanMacchia I have checked your homework and left some small comments; other katas are okay.

File: homeworks/german-macchia/06-basics-1/basics-1.js [hunk 2]
```diff
@@ -0,0 +1,415 @@
+// katas 1 https://www.codewars.com/kata/opposite-number
+function oppositeNumber(number) {
+  return number * -1;
+}
+
+// katas 2 https://www.codewars.com/kata/basic-mathematical-operations
+function mathematicOperation(sign, number1, number2) {
+  return eval(number1 + sign + number2);
```
[the remark this answers is not in the document]
Comment (GermanMacchia) [reply homeworks/german-macchia/06-basics-1/basics-1.js:8]: Yes! Thanks Vitalik. As I never used before it seem interesting to put it in practice. I've read the warning so I would avoid it from now on.

File: homeworks/german-macchia/06-basics-1/basics-1.js (hunk 1 above)
Comment (GermanMacchia) [reply homeworks/german-macchia/06-basics-1/basics-1.js:267]: checked!

File: homeworks/german-macchia/06-basics-1/basics-1.js [hunk 3]
```diff
@@ -0,0 +1,415 @@
...
+  if (days >= MIN_DAYS && days < MAX_DAYS) {
+    cost -= MIN_DISCOUNT;
+  } else if (days >= MAX_DAYS) {
+    cost -= MAX_DISCOUNT;
+  }
+
+  return cost;
+}
+
+// katas 5 https://www.codewars.com/kata/calculating-with-functions
```
[the remark this answers is not in the document]
Comment (GermanMacchia) [reply homeworks/german-macchia/06-basics-1/basics-1.js:34]: I see! I realiced about my mistake. I was thinking the function as if params were mandatory (like a JS function). But if we held no error handling we could use it as optional. I will correct this exercise and push a new version. Thanks Vitalii!

**Comment by VitalikMelnyk** [comment]
Well Done!
I will check other your PRs soon because have a lot of work).

**Review by VitalikMelnyk — APPROVED** [review]

**Comment by GermanMacchia** [comment]
> Well Done! I will check other your PRs soon because have a lot of work).

No problem! Thanks!

**Pull request merged**